### PR TITLE
Add `examples/semaphore.sh`

### LINKED ISSUE #13: example semaphore.sh & documentation
There is now 2 git repositories using semaphore.sh:
- https://github.com/weaveworks/tcptracer-bpf/blob/master/semaphore.sh
- https://github.com/iovisor/gobpf/pull/16/files

Instead of just including the script in README, we should probably have an examples/semaphore.sh file here so other projects could easily copy & adapt it.

It should also explain how to configure SemaphoreCI. See also https://github.com/weaveworks/tcptracer-bpf/issues/11

/cc @schu

### PR DESCRIPTION
**Pull request by schu**
Also, update the README: point to `examples/semaphore.sh` and explain
necessary configuration for Semaphore CI.

Fix #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,34 +49,31 @@ Our builds can be fetched from `kinvolk.io/aci/rkt/stage1-kvm`, e.g.
...
-## Usage on Semaphore CI
+## Using custom built stage1-kvm images on Semaphore CI
 
-As Semaphore CI doesn't include rkt by default on their platform, rkt must be
-downloaded as a first step.
+See `examples/semaphore.sh` for an example script which shows the necessary
+steps to use a custom stage1-kvm image and serves as a starting point.
 
-Example `semaphore.sh`:
+### Configuration
```
Comment (alban) [inline README.md:67]: the platform must be configured as well: by default it does not use the platform marked with "docker support"

Comment (schu) [reply README.md:67]: Right, thanks, updated.

File: README.md [hunk 2]
```diff
@@ -49,34 +49,35 @@ Our builds can be fetched from `kinvolk.io/aci/rkt/stage1-kvm`, e.g.
 rkt image fetch --insecure-options=image kinvolk.io/aci/rkt/stage1-kvm:1.23.0,kernelversion=4.9.6
```
Comment (alban) [inline README.md:49]: while you are at it, I would like if there was an example of "rkt run ..... uname -a" command directly in the readme

Comment (schu) [reply README.md:49]: Done.

**Pull request merged**